packages/php-config-printer/src/ServiceOptionConverter/BindAutowireAutoconfigureServiceOptionKeyYamlToPhpFactory.php

```diff
@@ -29,9 +29,9 @@ public function decorateServiceMethodCall($key, $yaml, $values, MethodCall $meth
             $method = 'share';
         }
 
-        if ($yaml === false) {
+        if (is_bool($yaml)) {
```

Comment:
Here the strict copmare would be better as there is only 1 value to pass and then easier to control

Reply:
Changed.